requirements-dev.txt

```diff
@@ -15,3 +15,5 @@ reno>=3.4.0
 ddt>=1.2.0,!=1.4.0
 matplotlib>=3.3
 seaborn>=0.9.0
+qiskit_sphinx_theme>=1.10
```

Comment:
Do we still need the sphinx-rtd-theme above? Similarly do we need `sphinx-automodapi` and `sphinx-tabs` anymore? I don't see modapi in the extension list in conf.py and also while sphinx-tabs is in the list I don't think anythin in the docs source is using it.

Reply:
That's true.
- `sphinx-rtd-theme`
- `sphinx-automodapi`, and 
- `sphinx-tabs`
are not necessary now.